tests/virt_autotest/reboot_and_wait_up_normal.pm

```diff
@@ -14,11 +14,22 @@ use strict;
 use warnings;
 use testapi;
 use base "reboot_and_wait_up";
+use virt_utils 'is_installed_equal_upgrade';
+use Utils::Backends 'is_remote_backend';
+use ipmi_backend_utils;
 
 sub run {
     my $self    = shift;
     my $timeout = 180;
-    $self->reboot_and_wait_up($timeout);
+
+    #online upgrade actually
+    if (is_remote_backend && check_var('ARCH', 'aarch64') && is_installed_equal_upgrade) {
```

Comment:
```suggestion
    return if (is_remote_backend && check_var('ARCH', 'aarch64') && is_installed_equal_upgrade);
    $self->reboot_and_wait_up($timeout);
```

Reply:
but it is just another format of same thing you can leave as it is now . just remove extra empty line in 31

Reply:
Done